next_generation/Dockerfile.sle15

```diff
@@ -0,0 +1,11 @@
+FROM registry.suse.de/home/yeey/pintserverng/containers/pint/pint-server-lambda-function:latest
```

Comment:
This is the internal repo? It may be an issue referencing a private registry from a public repository. I dont have an alternative suggestion though

Reply:
I don't think we need to reference it from ECR. We'll upload the image directly to ECR for production deployment. Dockerfile.sle15 is used for internal development only, for instantly test code changes before updating the package in IBS.